### ISSUE TITLE
Cannot see first commit when commit details are open

### ISSUE THREAD

**Issue by MansurAliKoroglu** [DonJayamanne/gitHistoryVSCode#583]
VSCode version: 1.49.1
Extension version : 0.6.9
Operating System: Win10 64bit

Steps:

Git: View History command
Click on a commit to open details
Scroll the very bottom of git history page
See the details blocking the view of first a few commit

![image](https://user-images.githubusercontent.com/43291306/93972287-a6882880-fd7a-11ea-89f4-250ed83c6c93.png)

**Comment by ole1986** [comment]
Well this is an error by design I would say. Since the "Commit Details" view is an overlay of the list view.

@DonJayamanne do you think this needs to be changed?

**Comment by MansurAliKoroglu** [comment]
Yes it's not that important but so far it was very inconvenient for me. We can just postpone it to implement a solution in a free time.
